test/fixtures/parser/ansi/update.yml

```diff
@@ -1,35 +0,0 @@
-# YML test files are auto-generated from SQL files and should not be edited by
```

Comment:
What's this about? Related to the PR?

Reply:
Nope. Must have committed mid YAML generation! It's been reverted.